src/appleseed.shared/application/progresstilecallback.cpp

```diff
@@ -64,9 +64,8 @@ namespace
       : public TileCallbackBase
     {
       public:
-        explicit ProgressTileCallback(Logger& logger)
-          : m_logger(logger)
-          , m_rendered_pixels(0)
+        explicit ProgressTileCallback()
```

Comment:
It was correct before, appleseed.shared shouldn't depend on the renderer's logger (`RENDERER_LOG_*`) which as its name implies, is specific to the renderer. Revert these changes.

In appleseed.renderer and appleseed.studio, you can use `renderer::global_logger()` (declared in `renderer/global/globallogger.h`) to access the renderer's global logger.